Exoft.Gamification.Api.Common/Models/ResponseModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Exoft.Gamification.Api.Common.Models
+{
+    public class ResponseModel
```

Comment:
have you used it any where?
also it should be abstract, and IsSuccessful should be set via constructor, and have only get method